src/Workspaces/CSharp/Portable/Extensions/OperatorPrecedence.cs

```diff
@@ -19,6 +19,7 @@ internal enum OperatorPrecedence
         Equality,
         RelationalAndTypeTesting,
         Shift,
+        Range,
```

Comment:
Note for @dotnet/roslyn-ide @sharwell This may have an impact on https://github.com/dotnet/roslyn/pull/21316

Reply:
Also note that precedence is still very up-in-the-air (its current location is very arbitrary), as is whether we'll have one or two range operators.